creator-node/scripts/updateContentBlacklist.js

```diff
@@ -0,0 +1,157 @@
+const axios = require('axios')
+const Web3 = require('web3')
+const web3 = new Web3()
+
+const models = require('../src/models')
+// const config = require('../src/config')
+const { sortKeys } = require('../src/apiHelpers')
+
+// node updateContentBlacklist.js <action> <type> <id>
+// node updateContentBlacklist.js add user 1
+// node updateContentBlacklist.js delete track 5
+
+// const PUBLIC_KEY = '0x8114bf583a3036dd4da30816003a37a4466ef817'
+const PRIVATE_KEY = '0x8b86274b0eef19dc09c0a4b72ccfb2fda73d5bc9d119fac10dde796602f2e3f3'
```

Comment:
you may want these as vars to pass into the script

Reply:
going to have these set as the env vars!